tests/checkpointing/test_trainer_checkpoint.py

```diff
@@ -75,9 +75,6 @@ def validation_step(self, batch, batch_idx):
         results.append(deepcopy(trainer.callback_metrics))
         best_model_paths.append(trainer.checkpoint_callback.best_model_path)
 
-    for idx in range(len(results) - 1):
-        assert results[idx]["val_loss"] > results[idx + 1]["val_loss"]
-
```

Comment:
Why is this removed? Does it not work anymore?

Reply:
No it doesn't.